internal/audit/audit.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package audit
+
+import (
+	"crypto/md5"
+	"fmt"
+	"github.com/juju/loggo"
+	"strings"
+)
+
+var logger = loggo.GetLogger("audit")
+
+
+type Operation string
+
+const (
+	SET_READ Operation = "set_read"
+	SET_WRITE Operation = "set_write"
+)
+
+func LogAcl(entity string, operation Operation, final_state []string) {
```

Comment:
Don't we also want to know what user is actually performing the given operation?